src/server/scripts/Northrend/VioletHold/boss_xevozz.cpp

```diff
@@ -208,6 +169,63 @@ class boss_xevozz : public CreatureScript
             if (victim->GetTypeId() == TYPEID_PLAYER)
                 Talk(SAY_SLAY);
         }
+
+        void SpellHit(Unit* who, const SpellInfo* spell) override
+        {
+            if (!who->ToCreature())
+                return;
+
+            if ((spell->Id == SPELL_ARCANE_POWER) || (spell->Id == H_SPELL_ARCANE_POWER))
+                Talk(SAY_SUMMON_ENERGY);
+        }
+
+        void UpdateAI(uint32 uiDiff) override
+        {
+            if (!UpdateVictim())
+                return;
+
+            events.Update(uiDiff);
+
+            if (me->HasUnitState(UNIT_STATE_CASTING))
+                return;
+
+            switch (uint32 eventId = events.ExecuteEvent())
```

Comment:
/home/jackpoz/trinity/sources/src/server/scripts/Northrend/VioletHold/boss_xevozz.cpp: In member function ‘virtual void boss_xevozz::boss_xevozzAI::UpdateAI(uint32)’:
/home/jackpoz/trinity/sources/src/server/scripts/Northrend/VioletHold/boss_xevozz.cpp:199:28: warning: unused variable ‘eventId’ [-Wunused-variable]
             switch (uint32 eventId = events.ExecuteEvent())
                            ^